data.json

```diff
@@ -730,7 +772,7 @@
                 "description": "Ask Me Anything about WebStorm",
                 "type": "forum",
                 "permissions": {
-                    "send_messages": ["JetBrains"]
+                    "send_messages_in_threads": ["JetBrains"]
```

Comment:
Support for `send_messages_in_threads` needs to be implemented? Maybe worth looking at genericifying the logic so that any key here will work?

Reply:
oh that's true, I thought I did that, well AI :D